.evergreen/podman-local-dev.sh

```diff
@@ -0,0 +1,44 @@
+#!/bin/bash
```

Comment:
This appears to be a copy of utils.sh. Am I right?

Reply:
Yep. Let me go ahead and... delete this ol' thing.